src/components/TaskListItem/TaskListItem.jsx

```diff
@@ -99,23 +99,19 @@ class TaskListItem extends React.Component {
               alt={repositoryOwner}
               className="task-list-item-repository-owner-avatar"
```

Comment:
Yikes, I didn't even see this one. Can we not eliminate the `task-list` and `item` from these, and have a more generic `repository-owner` and `user` avatar selector? We're not using them anywhere else (yet), but I doubt we want multiple ways of displaying these, and if we do, we can cross that bridge when we come to it.